website/content/docs/enterprise/license/utilization-reporting.mdx

```diff
@@ -117,7 +117,7 @@ reporting {
 ```
 
 If you need to, you can also opt out using an environment variable. This option
-requires a system restart. 
+requires to restart Vault. 
```

Comment:
Grammar nit: how do you feel about "a restart of the Vault service." instead of "to restart Vault."?